notebooks/RAG_evaluation.ipynb

```diff
@@ -0,0 +1,979 @@
+{
```

Comment:
<p>I think we can skip this cell, and assume that the user doesn't have a pregenerated evaluation dataset, this will also simplify a little bit the remaining code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>